.travis.yml

```diff
@@ -1,13 +1,41 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
+  - PYDATA=false CHECK_MIGRATION=true
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
+  - if [[ $TRAVIS_PULL_REQUEST != false ]]; then
+      git pull -q --no-edit origin develop;
+    fi;
+  - if [[ $PYDATA == true ]]; then
+      git pull -q --no-edit origin deploy-pydata;
```

Comment:
>What exactly is in deploy-pydata?

Adds `pydata` to INSTALLED_APPS and urlconfig. See the commit [cf8603aa](https://github.com/swcarpentry/amy/commit/cf8603aa5f6cc8fc33a829735e418ddb0e6b42dc)

>Are you sure git pull is the right way to go?

Isn't `pull` an aggregate command that executes `fetch` and `merge`. Shouldn't be a problem.

>Can we use git config --local instead of git config --global?

The instances destruct after every test. It does not make a difference.